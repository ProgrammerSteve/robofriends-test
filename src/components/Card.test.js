import {shallow,mount,render} from 'enzyme';
import Card from './Card.js';
import React from 'react';

// console.log(shallow(<Card/>));

// it('expects to render Card Component', ()=>{
//     expect(shallow(<Card/>).length).toEqual(1);
// })


it('expects to render Card Component', ()=>{
    expect(shallow(<Card/>)).toMatchSnapshot();
})