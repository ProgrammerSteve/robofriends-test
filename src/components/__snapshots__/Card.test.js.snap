// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render Card Component 1`] = `ShallowWrapper {}`;
